gslib/commands/version.py

```diff
@@ -117,7 +117,8 @@ def RunCommand(self):
           'gsutil path: {gsutil_path}\n'
           'compiled crcmod: {compiled_crcmod}\n'
           'installed via package manager: {is_package_install}\n'
-          'editable install: {is_editable_install}\n')
+          'editable install: {is_editable_install}\n'
+          'using gcloud storage: {is_shim_enabled}\n')
```

Comment:
I wonder if the field title should be "is shim enabled". Because enabling the shim does not guarantee that gcloud storage will be used, so will the user be misguided that the `gcloud storage` is being used for all the commands?

Reply:
Good call. I'll update.